# DP: Don't build the gcjwebplugin, even when configured with --enable-plugin

Index: b/src/libjava/configure.ac
===================================================================
--- a/src/libjava/configure.ac
+++ b/src/libjava/configure.ac
@@ -67,6 +67,8 @@ AC_ARG_ENABLE(browser-plugin,
      esac],
     [browser_plugin_enabled=no]
 )
+# FIXME: don't build the plugin, this option collides with GCC plugin support
+plugin_enabled=no
 
 AC_ARG_ENABLE(gconf-peer,
   AS_HELP_STRING([--enable-gconf-peer],
